config/routes.rb

```diff
@@ -418,6 +418,8 @@
   get "/:username/:view" => "stories#index",
       :constraints => { view: /comments|moderate|admin/ }
   get "/:username/:slug" => "stories#show"
+  get "/:sitemap" => "sitemaps#show",
```

Comment:
Google claims that the sitemaps more or less need to be on the root, so this establishes a constraint that `sitemap-` is reserved territory on the root.

Reply:
Won't this conflict with `get "/:username"`? Although tests passed 🤔 

I wonder if we can do something like `get "/sitemap-:sitemap_slug"` but probably not.

Reply:
The next line dictates after the `,` the constraint, that this rule only applies if `:sitemap` matches the `sitemap-` pattern.

Reply:
Ah right 😅